### PR TITLE
Adding launchpad network

### PR DESCRIPTION
**Pull request by TrevorBenson**
Adding config-launchpad.json and updating prereqs.sh to have additional network definition.

### CODE REVIEW

File: files/config-launchpad.json [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+  "ByronGenesisFile": "/opt/cardano/cnode/files/byron-genesis.json",
+  "ByronGenesisHash": "3815702729e4ba80ed3b6fc900be90a4b276daf5fcb4132c588cb038bff611f0",
+  "LastKnownBlockVersion-Alt": 0,
+  "LastKnownBlockVersion-Major": 4,
+  "LastKnownBlockVersion-Minor": 0,
+  "MaxKnownMajorProtocolVersion": 2,
+  "PBftSignatureThreshold": 0.9,
+  "Protocol": "Cardano",
+  "RequiresNetworkMagic": "RequiresMagic",
+  "ShelleyGenesisFile": "/opt/cardano/cnode/files/shelley-genesis.json",
```
Comment (rdlrt) [inline files/config-launchpad.json:13]: should be genesis.json

**Review by rdlrt — APPROVED** [review]

**Review by rdlrt — CHANGES_REQUESTED** [review]
Would be nice to update docs/basics.md too

**Comment by TrevorBenson** [comment]
@rdlrt fixed.

**Review by rdlrt — APPROVED** [review]

**Comment by rdlrt** [comment]
Cool Thanks

**Pull request merged**